src/redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const BooksPage = () => (
+  <>
+    <div className="books-container d-flex">
+      <h2>Book List</h2>
+      <button type="button">Remove</button>
```

Comment:
- [ ] As it is a requirement for this project(`see screenshot below`) and best practice, making sure building blocks of your app are re-usable components, please kindly add a `Book` component that; 
- Takes the book data as a prop and renders the title and the author
- Has a button for removing books(the button wouldn't remove the books *yet*, this is just a placeholder for now).
![re-usable1](https://user-images.githubusercontent.com/65852621/149915636-548b33f1-b549-42e0-936a-e4d10767f390.png)
![re-usable2](https://user-images.githubusercontent.com/65852621/149915674-fc40340e-d757-428a-87b8-8ab0b365c25a.png)